manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: LuaForWindows.rjpcomputing

```

Better to use publisher first
```suggestion
PackageIdentifier: rjpcomputing.LuaForWindows
```